tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1582,4 +1582,28 @@ class CompletionSuite extends BaseCompletionSuite {
     topLines = Some(1),
   )
 
+  check(
+    "for-comprehension",
+    """|object B {
+       |  val a = for {
+       |    foo <- List("a", "b", "c")
+       |    abc@@ = println("Print!")
+       |  } yield abc
+       |}
+       |""".stripMargin,
+    "",
+  )
+
+  check(
+    "for-comprehension2",
+    """|object B {
+       |  val a = for {
+       |    foo <- List("a", "b", "c")
+       |    abcCURSORde = println("Print!")
+       |  } yield abc@@
```

Comment:
I have now found another issue: `path` here is `Ident :: CaseDef :: Match`, so in this example it looks like `List(("a", ())).map{ case abc@@}`. I realised that for case completions on tuples, we are not checking if pattern (here `abc`) matches the completion (`($1)`)

Reply:
Yes, I already started looking into it. I will adjust test here so we can merge without waiting for the other pr